packages/expo-navigation-bar/src/NavigationBar.ts

```diff
@@ -139,219 +45,33 @@ export async function getBorderColorAsync(): Promise<string> {
  * ```
  * @param visibility Based on CSS visibility property.
  */
-export async function setVisibilityAsync(visibility: NavigationBarVisibility): Promise<void> {
-  if (__DEV__) {
-    potentiallyWarnAboutEdgeToEdge();
-  }
-  if (Platform.OS !== 'android') {
-    console.warn('`setVisibilityAsync` is only available on Android');
-    return;
-  }
-  await ExpoNavigationBar.setVisibilityAsync(visibility);
-}
-
-/**
- * Get the navigation bar's visibility.
- *
- * @example
- * ```ts
- * const visibility = await NavigationBar.getVisibilityAsync("hidden");
- * ```
- * @returns Navigation bar's current visibility status. Returns `hidden` on unsupported platforms (iOS, web).
- */
-export async function getVisibilityAsync(): Promise<NavigationBarVisibility> {
-  if (__DEV__) {
-    potentiallyWarnAboutEdgeToEdge();
-  }
-  if (Platform.OS !== 'android') {
-    console.warn('`getVisibilityAsync` is only available on Android');
-    return 'hidden';
-  }
-  return ExpoNavigationBar.getVisibilityAsync();
-}
-
-/**
- * Changes the navigation bar's button colors between white (`light`) and a dark gray color (`dark`).
- *
- * @example
- * ```ts
- * NavigationBar.setButtonStyleAsync("light");
- * ```
- * @param style Dictates the color of the foreground element color.
- */
-export async function setButtonStyleAsync(style: NavigationBarButtonStyle): Promise<void> {
-  if (__DEV__) {
-    potentiallyWarnAboutEdgeToEdge();
-  }
+export function setVisibilityAsync(visibility: NavigationBarVisibility) {
   if (Platform.OS !== 'android') {
-    console.warn('`setButtonStyleAsync` is only available on Android');
+    console.warn('`setVisibilityAsync` method is only available on Android');
```

Comment:
```suggestion
    console.warn('`setVisibilityAsync` is only available on Android');
```

for consistency with other messages like https://github.com/expo/expo/blob/main/packages/expo-status-bar/src/StatusBar.android.tsx#L85